### PR TITLE
Add o message

### LINKED ISSUE #51: Add o-message
Alerts have o-message styling (ish), and need to be removed once that component is complete and has an extra variant.

### PR DESCRIPTION
**Pull request by gvonkoss**
Adds an alert and a notice message to the component page, through `o-message`
Closes #51

### CODE REVIEW

File: src/main.scss [hunk 1]
```diff
@@ -1,9 +1,10 @@
 $o-buttons-is-silent: false;
 $o-forms-is-silent: false;
 $o-header-services-is-silent: false;
+$o-normalise-is-silent: false;
+// $o-message-is-silent: false;
```
Comment (JakeChampion) [inline src/main.scss]: Should this be uncommented?

Comment (gvonkoss) [reply src/main.scss]: Whooops. That just shouldn't be there!

File: bower.json [hunk 2]
```diff
@@ -8,18 +8,19 @@
   ],
   "homepage": "https://github.com/Financial-Times/origami-registry-ui",
   "dependencies": {
-    "o-header-services": "^2.0.5",
-    "o-table": "^6.5.0",
+		"o-autoinit": "^1.3.3",
+		"o-buttons": "^5.10.1",
+		"o-colors": "^4.1.5",
     "o-grid": "^4.3.7",
```
Comment (notlee) [inline bower.json]: Predantic: Looks like some funky indentation.

Comment (notlee) [reply bower.json]: Predantic: Predictably pedantic.

**Comment by notlee** [comment]
Oh! I take that back. Think o-message needs to be set to silent mode.

**Pull request merged**